### ISSUE TITLE
Module Manager error?

### ISSUE THREAD

**Issue by DruggedBunny** [blitz-research/monkey2#458]
I just clicked to install png2polygon from Ted2Go's Module Manager, and it appears to be rebuilding all modules, including SDL2, assimp, stb-vorbis, std, etc!

Guessing that's not meant to happen, certainly never done that before!

![image](https://user-images.githubusercontent.com/1781537/51339687-f92d1f80-1a84-11e9-9563-998d76ec634f.png)

**Comment by XANOZOID** [comment]
I have also noticed this and thought it may have been a bug . . . I wonder if this is an IDE or tool related thing.

**Comment by engor** [comment]
IDE update modules, don't rebuild from scratch.
I think proper way is to update only modules from `depends` section.

**Comment by blitz-research** [comment]
Yes, after a module manager update, monkey2 just does an 'update modules',
and the first time this is done all modules end up getting built.

> I think proper way is to update only modules from depends section.

Actually, I think just updating the installed/updated modules might be
enough as this should automatically update dependancies?

**Comment by DruggedBunny** [comment]
Is that new? I know in the past (quite some time ago, admittedly) it didn't seem to do this. I actually thought it might be weirdly related to the rebuildall*.bat changes recently, eg. running updateall in error.

I didn't want to cancel the build, not knowing what state things would have been left in, and basically had to find something else to do for half an hour or so! (#firstworldproblems)

It's hard to see the need to rebuild std-vorbis or assimp when adding png2polygon, for instance...

**Comment by engor** [comment]
> Actually, I think just updating the installed/updated modules might be
> enough as this should automatically update dependancies?

OK.

> I didn't want to cancel the build, not knowing what state things would have been left in, and basically had to find something else to do for half an hour or so! (#firstworldproblems)

Yes, if you use precompiled version (and many of users use that) then first update is the same as total rebuild coz there is no built files yet.

**Comment by engor** [comment]
I just pushed fix into ted2go's dev-branch, since now it will update only performed modules.

**Comment by seyhajin** [comment]
Can be closed?
